lib/l10n/intl_messages.arb

```diff
@@ -1226,12 +1226,6 @@
     "placeholders_order": [],
     "placeholders": {}
   },
-  "nameToBeDisplayed": "Name to be displayed to recipients",
-  "@nameToBeDisplayed": {
-    "type": "text",
-    "placeholders_order": [],
-    "placeholders": {}
-  },
```

Comment:
why do we need delete it?

Reply:
oh you are right, i rebased this file wrong...